akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphStageTimersSpec.scala

```diff
@@ -39,13 +39,17 @@ class GraphStageTimersSpec extends AkkaSpec {
   implicit val mat = ActorMaterializer()
 
   class TestStage(probe: ActorRef, sideChannel: SideChannel) extends SimpleLinearGraphStage[Int] {
-    override def createLogic = new SimpleLinearStageLogic {
+    override def createLogic(inheritedAttributes: Attributes) = new TimerGraphStageLogic(shape) {
       val tickCount = Iterator from 1
 
       setHandler(in, new InHandler {
         override def onPush() = push(out, grab(in))
       })
 
+      setHandler(out, new OutHandler {
```

Comment:
I removed `SimpleLinearStageLogic` because I extracted timer related stuff to the subclass `TimerGraphStageLogic`